src/types/Story.ts

```diff
@@ -187,7 +187,8 @@ export namespace Story {
      * @deprecated Please use `Status` instead.
      * @see Status
      */
-    export import LifecycleStatus = Status;
+    export const LifecycleStatus = Status;
+    export type LifecycleStatus = Status;
```

Comment:
Great! Now let's do this for other `export import` statements in this repo. :+1:

Reply:
Done: https://github.com/prezly/javascript-sdk/pull/307/commits/12ed5985240b5c3021e14762c6499d8299c26a9a